.github/workflows/ci.yaml

```diff
@@ -43,9 +43,12 @@ jobs:
         with:
           channels: "${{ env.CHANNELS }}"
       - name: Build Nivlheim
-        run: >
-          guix build nivlheim
-          --with-source=nivlheim=$PWD
+        run: |
+          VERSION=`cat VERSION`
+          if [[ "$GITHUB_REF" != "refs/heads/master" ]]; then
+            VERSION="$VERSION-$GITHUB_REF_NAME"
+          fi
+          guix build nivlheim --with-source=nivlheim@$VERSION=$PWD
```

Comment:
`--with-source=nivlheim@$VERSION` vil ikke fungere, da Guix kun kjenner til én versjon av Nivlheim (2.7.10 atm).  Så denne transformasjonen har ingen effekt (Guix skal printe en melding om det), med det resultat at jobben bygger versjonen av Nivlheim som er hardkodet i Guix i stedet for lokal checkout.

Et par eksempler for å vise hvordan transformasjoner fungerer:

* bygge Nivlheim (og alle dependencies) med en patchet Go:
  * `guix build nivlheim --with-source=go@1.16=$HOME/my-go-checkout`
* bygge mreg (og alle dependencies) med Django 2.2 i stedet for 3.2:
  * `guix build mreg --with-input=django@3.2=django@2.2`

Reply:
Vi trenger å kunne bygge siste commit i en gitt branch med versjonen som står i filen VERSION pluss suffiks "-branchnavn" hvis ikke branchen er master.

Reply:
CI-jobben feilet fordi `guix-install-action` feilet, fordi Guix-infrastrukturen er delvis nede...  Prøver å restarte den i morgen.

Reply:
Funker nå.  `branch`-variabelen må justeres til å ta høyde for tags, skal fikse litt senere.